### PR TITLE
Trying to fix geometry tests

### PR DESCRIPTION
**Pull request by OndrejSpanel**
This PR starts effort to fix geometry tests. The commit fixes an obvious ommission - the required `path` parameter was not provided for `TubeGeometry`. The test executes now a bit more, but it still does not pass, failed parts are:

> 4. properties are equal
> 23. geometry matching source uuid not in output
 
The second failure can be found in some other geometry tests.

If someone can give an advice how to fix them, I will be happy to continue with the fix. I will try to continue anyway, but without advice I am afraid it will require a bit more effort.

### CODE REVIEW

File: test/unit/src/geometries/TubeGeometry.tests.js [hunk 1]
```diff
@@ -8,17 +8,20 @@ import {
...
+
 export default QUnit.module( 'Geometries', () => {
 
 	QUnit.module( 'TubeGeometry', ( hooks ) => {
 
 		var geometries = undefined;
 		hooks.beforeEach( function () {
 
-			const parameters = {};
+            var path = new LineCurve3(new Vector3(0, 0, 0), new Vector3(0, 1, 0));
```
Comment (Mugen87) [inline test/unit/src/geometries/TubeGeometry.tests.js]: Please improve the code style in your PRs. Als ensure to use hard tabs and not soft tabs. More information here:

https://github.com/mrdoob/three.js/wiki/How-to-contribute-to-three.js

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by OndrejSpanel** [comment]
I hope I have fixed all code style issues.

**Comment by Mugen87** [comment]
Much better 👍 . Please check #12778, too.

File: test/unit/src/geometries/ShapeGeometry.tests.js [hunk 2]
```diff
@@ -8,17 +8,22 @@ import {
 	ShapeBufferGeometry
 } from '../../../../src/geometries/ShapeGeometry';
 
+import {Shape} from '../../../../src/extras/core/Shape';
```
Comment (mrdoob) [inline test/unit/src/geometries/ShapeGeometry.tests.js]: Minor thing...

```js
import { Shape } from '../../../../src/extras/core/Shape';
```

File: test/unit/src/geometries/TubeGeometry.tests.js [hunk 3]
```diff
@@ -8,17 +8,20 @@ import {
 	TubeBufferGeometry
 } from '../../../../src/geometries/TubeGeometry';
 
+import {LineCurve3} from '../../../../src/extras/curves/LineCurve3'
+import {Vector3} from '../../../../src/math/Vector3'
```
Comment (mrdoob) [inline test/unit/src/geometries/TubeGeometry.tests.js]: Same here...

```js
import { LineCurve3 } from '../../../../src/extras/curves/LineCurve3'
import { Vector3 } from '../../../../src/math/Vector3'
```

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**